### PR TITLE
test functions

### PR DESCRIPTION
**Pull request by AdrianSalas1992**

### CODE REVIEW

**Comment by DiegoE-prog** [comment]
Please add your reviewers.

**Review by JAMendozaSa — CHANGES_REQUESTED** [review]
Erase de new project please

File: frontend/jce-app/my-app/package.json
Comment (MauricioJChavez) [inline frontend/jce-app/my-app/package.json:1]: Everything related to the new application that you created must be removed first, as Juan said.

**Review by MauricioJChavez — CHANGES_REQUESTED** [review]
Remove the new project created inside the {frontend/jce{-app folder

**Pull request closed without merging**